### ISSUE TITLE
lit-plugin extension breaks vscode auto completion

### ISSUE THREAD

**Issue by Kokujou** [runem/lit-analyzer#201]
i had this problem for a while now and figured out it's this extension causing the issue
usually when you type in VSCode you get a suggestion for what function you might mean.
nowadays vscode not only suggests you function from currently not imported files, it also adds an import statement if you select the suggestion
but if this plugin is installed this functionality breaks.

i didn't configure anything for this extension, so i'm basically clean. please help me!

**Comment by Artur-** [comment]
This is the same as #200

**Comment by Kokujou** [comment]
you can say so. only different thing is that this issue also occurs with javascript code.

**Comment by 43081j** [comment]
Yep, duplicate of #220, the upcoming fix should solve both cases.
